lib/player.rb

```diff
@@ -0,0 +1,18 @@
+class Player
+
+  attr_reader :gender
+
+  def initialize gender
```

Comment:
Consider the class method way I demod for setting the gender. Another developer might accidentally pass a string instead of a symbol here. Having a class method, `.male` that instantiates a male player would sort of solve this problem. Alternatively you could have a shallow inheritance hierarchy:

```ruby
class Player

   def self.male
     Male.new
   end

    def self.female
     Female.new
   end

   class Male < Player
     
      def male?
         true
      end

  end

  class Female < Player

    def male?
      false
    end

  end

  # player methods common to both genders...

end